src/include/conversions.h

```diff
@@ -61,8 +61,8 @@ as_status as_udf_file_to_pyobject(as_error *err, as_udf_file *entry,
 as_status as_udf_files_to_pyobject(as_error *err, as_udf_files *files,
                                    PyObject **py_files);
 
-as_status strArray_to_py_list(as_error *err, int num_elements, int element_size,
-                              char str_array_ptr[][element_size],
+as_status strArray_to_py_list(as_error *err, int num_elements,
+                              char str_array_ptr[][AS_ROLE_SIZE],
```

Comment:
Why was this change made? Is this function only ever used for roles? If so I think it should be renamed.